index.html

```diff
@@ -0,0 +1,231 @@
+        <div class="middle-section-container2-1">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">Professional Chef Program</h3>
+          <p class="middle-section-para">
+            The Professional Chef Program is a 12-month, full-time program that
+            will teach you the culinary skills and business knowledge you need
+            to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-2">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Pastry Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Pastry Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-3">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Culinary Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Culinary Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+      </div>
+
+      <button type="button" class="btn">See the full program</button>
+    </section>
+    <div class="teacher-topic">
+      <h2 class="teacher">Featured Teachers</h2>
+    </div>
+    <div class="container"></div>
+
+    <section class="partner-section">
+      <h2 class="partner">Our Partners</h2>
+      <div class="line"></div>
+      <ul class="partner-elments">
+        <li>
+          <img class="partner-img" src="google.png" alt="partner1" />
+        </li>
+        <li>
+          <img class="partner-img" src="airbnb.png" alt="partner1" />
+        </li>
+        <li>
+          <img class="partner-img" src="apple.png" alt="partner1" />
+        </li>
+        <li>
+          <img class="partner-img" src="youtube.png" alt="partner1" />
+        </li>
+      </ul>
+    </section>
+
+    <footer id="join" class="footer-section">
+      <h2 class="footer-heading">Contact Us</h2>
+      <div class="line"></div>
+        <div class="bottom-footer">
+        <img src="logo.svg" alt="logo" class="logo-img-footer" />
+        <p class="footer-para1">
+          For any questions or concerns, please contact us at:
+        </p>
+      </div>
+      <p class="footer-para">david630@gmail.com</p>
+      <p class="footer-para">+56 9 735 44953</p>
+      <p class="footer-info">
+        Webiste and Brand Design by David Studio | Santiago, Chile.<br />
+        Wesite Design by David Underwood 2022 Feb
+      </p>
+    </footer>
```

Comment:
- [ ] [OPTIONAL] Good job so far! You've pretty much covered what the previous code reviewer said, but it looks like the `spacing` and `placement` of your `Footer section` elements are messed up. I have highlighted them in the image below:

![image](https://user-images.githubusercontent.com/63915024/191834999-59114423-12e4-4220-8c4a-aa85b9f58f00.png)